src/PolyhedralGeometry/triangulations.jl

```diff
@@ -254,3 +254,130 @@ A polyhedron in ambient dimension 8
+
+julia> S = SubdivisionOfPoints(C, Incidence)
+A subdivision of points in ambient dimension 2
+```
+"""
+SubdivisionOfPoints(P::Polyhedron, Incidence::IncidenceMatrix) = SubdivisionOfPoints(point_matrix(vertices(P)), Incidence)
+
+
+@doc Markdown.doc"""
+    SubdivisionOfPoints(P::Polyhdron, Weights::AbstractVector)
+
+# Arguments
+- `P::Polyhedron`: A polyhedron whose vertices are the points of the subdivision.
+- `Weights::AbstractVector`: A vector with one entry for every point indicating the height of this point.
+
+A subdivision of points formed by placing every vertex of `P` at the corresponding
+height, then taking the convex hull and then only considering those cells
+corresponding to faces visible from below ("lower envelope").
+
+# Examples
+Compute a triangulation of the square
+```jldoctest
+julia> C = cube(2);
+
+julia> Weights = [0,0,1,2];
+
+julia> S = SubdivisionOfPoints(C, Weights)
+A subdivision of points in ambient dimension 2
+```
+"""
+SubdivisionOfPoints(P::Polyhedron, Weights::AbstractVector) = SubdivisionOfPoints(point_matrix(vertices(P)), Weights)
+
+
+SubdivisionOfPoints(P::Polyhedron, MaximalCells::Vector{Vector{Int64}}) = SubdivisionOfPoints(point_matrix(vertices(P)), IncidenceMatrix(MaximalCells))
+SubdivisionOfPoints(Iter::SubObjectIterator{<:PointVector}, Incidence::IncidenceMatrix) = SubdivisionOfPoints(point_matrix(Iter), Incidence)
+SubdivisionOfPoints(Iter::SubObjectIterator{<:PointVector}, Weights::AbstractVector) = SubdivisionOfPoints(point_matrix(Iter), Weights)
+SubdivisionOfPoints(Iter::SubObjectIterator{<:PointVector}, MaximalCells::Vector{Vector{Int64}}) = SubdivisionOfPoints(point_matrix(Iter), IncidenceMatrix(MaximalCells))
+
+
+
+
+@doc Markdown.doc"""
+    gkz_vector(SOP::SubdivisionOfPoints)
+
+Compute the gkz vector of a triangulation given as a subdivision of points, SOP.
+More generally, when SOP is not a triangulation, compute the vector `v` whose
+i-th coordinate is the sum of the normalized volumes of all maximal cells
+containing the i-th point.
+
+# Examples
+Compute the gkz vector of one of the two regular triangulations of the square.
+```jldoctest
+julia> C = cube(2);
+
+julia> Triang = SubdivisionOfPoints(C,[[1,2,3],[2,3,4]])
+A subdivision of points in ambient dimension 2
+
+julia> gkz_vector(Triang)
+4-element Vector{Int64}:
+ 4
+ 8
+ 8
+ 4
+```
+"""
+function gkz_vector(SOP::SubdivisionOfPoints)
+    gkz = []
+    #after coding points as SubObjectIterator, take
+    #    point_mat = point_matrix(points(SOP))
+    #instead
+    point_mat = hcat(points(SOP)...)'
+    for i in 1:length(points(SOP))
+        sum_of_volume = 0
+        #maximal_cells are given as Sets, so we collect to make them vectors.
+        for triangle ∈ filter(x->i ∈ x, collect(maximal_cells(SOP)))
+            sum_of_volume += normalized_volume(convex_hull(point_mat[Vector(triangle),:]))
+        end
+        push!(gkz,sum_of_volume)
+    end
+    Vector{Int64}(gkz)
+end
```

Comment:
I have several questions here:
- This only works for triangulations, but there is no check that this is a triangulation.
- polymake can compute gkz-vectors, why not use this?
- Why are the resulting numbers integers? I can also triangulate vertices of some non-lattice polytope.

In general doing a 
- convex hull
- normalized volume
computation is overkill here. Computing volumes of simplices should just be determinants.

There is `gkz_vector` in application polytope, but it is not canonically accessible via Polymake.jl. In polymake it works as follows:
```
polytope > $c = cube(2,1,0);

polytope > $V = $c->VERTICES;

polytope > $T = new Array<Array<Int>>([[0,1,2],[1,2,3]]);

polytope > print $V;
1 0 0
1 1 0
1 0 1
1 1 1

polytope > print $T;
0 1 2
1 2 3

polytope > print gkz_vector($V, $T);
1 2 2 1
```
Maybe @benlorenz knows a way to get at this directly?

Reply:
1) One can generalize the notion of a gkz vector to any subdivision: the i-th coordinate is the sum of the volumes of the polyhedra containing the i-th point.

2) I did not use polymake's `gkz_vector` because I couldn't parse the documentation to find out how. I am happy to switch this!

3) Good call. There is no reason for the output to be integers. I will fix this too. 

4) I agree, determinants are quicker. And sure enough this function is very slow. The only upside is that it produces a vector for any subdivision of points. Maybe @micjoswig knows whether or not such a generalization is even remotely desirable here.

Reply:
```
julia> c = cube(2,0,1)
A polyhedron in ambient dimension 2

julia> V = c.pm_polytope.VERTICES
pm::Matrix<pm::Rational>
1 0 0
1 1 0
1 0 1
1 1 1


julia> T = IncidenceMatrix([[1,2,3],[2,3,4]])
2×4 IncidenceMatrix
[1, 2, 3]
[2, 3, 4]


julia> TT = [Polymake.to_zero_based_indexing(Polymake.row(T,i)) for i in 1:Polymake.nrows(T)]
2-element Vector{Polymake.SetAllocated{Int64}}:
 pm::Set<long, pm::operations::cmp>
{0 1 2}
 pm::Set<long, pm::operations::cmp>
{1 2 3}

julia> Polymake.call_function(:polytope, :gkz_vector, V, TT)
pm::Vector<pm::Rational>
1 2 2 1
```
This seems to work.